contrib/cirrus/prebuild.sh

```diff
@@ -44,10 +44,10 @@ get_env_key() {
 }
 
 # Only need to check CI-stuffs on a single build-task, there's only ever
-# one prior-fedora task so use that one.
+# one current latest fedora task so use that one.
```

Comment:
This is (temporarily) no longer true. We now run this check twice. I don't see a way to run on just one of the current-fedora jobs so  I think this is okay. The `dnf install` is absolutely NOT okay, and I have made a note to fix that in automation_images. For now it needs to stay.

My tentative suggestion is to fix this comment, remove the "only ever one" part, maybe acknowledge that usually this will only be one task but sometimes two. WDYT? Any better ideas?

Reply:
> The dnf install is absolutely NOT okay, and I have made a note to fix that in automation_images. For now it needs to stay.

Yes agree I made a not to myself too, adding one package there is simple enough

> My tentative suggestion is to fix this comment, remove the "only ever one" part, maybe acknowledge that usually this will only be one task but sometimes two. WDYT? Any better ideas?

Yes sounds good.